python/dlr/tf2_model.py

```diff
@@ -50,6 +83,14 @@ def __init__(self, model_path, dev_type=None, dev_id=None, error_log_file=None,
         outputs_tensor_info = meta_graph_def.signature_def[signature_def_key].outputs
         self.input_tensor_names = list(inputs_tensor_info.keys())
         self.output_tensor_names = list(outputs_tensor_info.keys())
+        self.num_inputs = len(self.input_tensor_names)
+        self.num_outputs = len(self.output_tensor_names)
+        self.input_dtypes = _convert_dtypes(
```

Comment:
We can use `tf.dtypes.DType` to convert type_id to string
```
input_dtypes=[types_pb2.DT_FLOAT, types_pb2.DT_FLOAT, types_pb2.DT_INT16]
[tf.dtypes.DType(a).name for a in input_dtypes]
['float32', 'float32', 'int16']
```
https://www.tensorflow.org/versions/r2.4/api_docs/python/tf/dtypes/DType

Reply:
Updated!